backend/plugins/authorization.js

```diff
@@ -0,0 +1,68 @@
+var errors = require('restify-errors');
+
+import {getSetting} from '../settings.js';
+import {generateAndSaveAccessToken} from '../utils/authUtils';
+import fetch from 'node-fetch';
+
+const ESCAPED_ROUTES = [
+  new RegExp('^/$'),
+  new RegExp('^/login$'),
+  new RegExp('^/static/'),
+  new RegExp('^/oauth2')
+]
+
+function accessTokenIsValid(access_token) {
+    return getSetting('ACCESS_TOKEN') === access_token;
+}
+
+let win = '';
+
+export function PlotlyOAuth(electronWindow) {
+    win = electronWindow;
```

Comment:
🐮  can we remove this assignment? if it wasn't passed in, then it'll just be undefined and so the `if (electronWindow)` check should still work as intended